Formula/cloudstate-cli.rb

```diff
@@ -0,0 +1,14 @@
+class CloudstateCli < Formula
+  desc "Cloudstate CLI"
+  homepage "https://github.com/sleipnir/cloudstate-cli"
+  url "https://github.com/sleipnir/cloudstate-cli/releases/download/0.3.54/cloudstate-0.3.54-osx.tar.gz"
```

Comment:
Please don't submit your own software: https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

Reply:
CloudState (https://cloudstate.io/) is a serveless platform for stateless function building. With a rapidly growing and evolving community. Cloudstate CLI is part of the project and is not just my contribution but the entire Cloudstate community

Reply:
That's nice but you're the owner of the repo. Making this self-submitted software.

Reply:
Understand. Ok. Thank you.